repos/fsharp.proj

```diff
@@ -1,37 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
-  <PropertyGroup>
-    <PackagesOutput>$(ProjectDirectory)/BuildFromSource/Release/bin</PackagesOutput>
-    <BuildRevision>181017</BuildRevision>
-    <BuildCommand>$(ProjectDirectory)/src/buildfromsource$(ShellExtension)</BuildCommand>
-    <RepoApiImplemented>false</RepoApiImplemented>
-    <!-- This repo is not currently built in ProdCon -->
-    <OrchestratedManifestBuildName>N/A</OrchestratedManifestBuildName>
-    <OfficialBuildId>20190131.4</OfficialBuildId>
-  </PropertyGroup>
-
-  <!-- Auto-detect the root NuGet.Config, but also update the build-from-source-specific one. -->
-  <ItemGroup>
-    <NuGetConfigFiles Include="$(ProjectDirectory)/src/buildfromsource/NuGet.config" />
-  </ItemGroup>
 
   <PropertyGroup>
-    <PathListSeparator>:</PathListSeparator>
-    <PathListSeparator Condition="'$(OS)' == 'Windows_NT'">%3B</PathListSeparator>
+    <BuildCommandArgs>--restore --build --pack --publish</BuildCommandArgs>
+    <BuildCommandArgs>$(BuildCommandArgs) --configuration $(Configuration)</BuildCommandArgs>
+    <BuildCommandArgs>$(BuildCommandArgs) --binaryLog</BuildCommandArgs>
+    <BuildCommandArgs>$(BuildCommandArgs) /p:DotNetBuildFromSource=true</BuildCommandArgs>
```

Comment:
I presume I can delete this line because of [this](https://github.com/dotnet/source-build/blob/316f301e1a1a73cc14c69f3d7fa5dabff6bb6a2c/repos/dir.props#L58)?

Reply:
Yeah, that should be fine, F# isn't doing any wonky docker stuff or anything I don't think.